src/Tiqr/TiqrConfiguration.php

```diff
@@ -74,6 +74,8 @@ public function __construct(array $tiqrConfiguration)
             $this->options['firebase.credentialsFile'] = $tiqrConfiguration['library']['firebase']['credentialsFile'];
             Assertion::boolean($tiqrConfiguration['library']['firebase']['cacheTokens']);
             $this->options['firebase.cacheTokens'] = $tiqrConfiguration['library']['firebase']['cacheTokens'];
+            Assertion::string($tiqrConfiguration['library']['firebase']['tokenCacheDir']);
```

Comment:
This is not type safe enough also see the failing PHPStan

Reply:
Added check if it is an array

Reply:
We probably want to convert this to an object instead of an array when there's maintenance.